packages/subiquity_client/lib/src/types.dart

```diff
@@ -1,3 +1,6 @@
+// TODO(Lukas): Rename enums to dart style
+// ignore_for_file: constant_identifier_names
+
```

Comment:
Could we also ignore `always_put_required_named_parameters_first` in this file and move the `sizingPolicy` back? Otherwise we'd need update generator.py accordingly.

Reply:
Does the generator not taking into consideration that these are named arguments and not positional ones?

Reply:
The generator is really simple: it traverses subiquity's types.py and generates Dart code with all parameters in the same order. It marks a parameter as `required` if the corresponding default value is `None` or something that can't be converted into a primitive Dart type.

Reply:
Ah, makes sense, I'll revert that change and add an ignore!

Reply:
Thank you!